web/js/map-OpenLayers.js

```diff
@@ -121,7 +113,8 @@ var fixmystreet = fixmystreet || {};
         return fixmystreet.markers.getFeaturesByAttribute('id', problem_id)[0];
       },
 
-      marker_size_for_zoom: function(zoom) {
+      marker_size_for_zoom: function() {
```

Comment:
This function is overridden in the two wmts* files, so you'll need to change their behaviour too. Or have a shared function looking up the zoom calling different functions.